docs/source/usage/extensions.rst

```diff
@@ -1,36 +1,36 @@
 Extensions
 ==========
-MetaSynth is developed with a strong focus on extensibility, allowing developers to easily build upon its existing functionality. This page aims to document major extensions that are currently available.
+Metasyn is developed with a strong focus on extensibility, allowing developers to easily build upon its existing functionality. This page aims to document major extensions that are currently available.
 
 .. note:: 
-    Refer to our :doc:`/developer/developer` for more information on how to develop (additional functionality) for MetaSynth.
+    Refer to our :doc:`/developer/developer` for more information on how to develop (additional functionality) for metasyn.
 
 Disclosure control
 ------------------
-`Disclosure Control <https://github.com/sodascience/metasynth-disclosure-control>`_ is a plugin developed in-house for MetaSynth.
+`Disclosure Control <https://github.com/sodascience/metasyn-disclosure-control>`_ is a plugin developed in-house for metasyn.
```

Comment:
This link is currently also broken, we should not forget to rename the disclosure control package after merging this PR.

Reply:
Thanks, I already have a rename PR for that package as well, but I haven't renamed the repo yet.